evals/cybench/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
```

Comment:
If the idea is that the user builds the images locally, do we need to add the `x-local` flag to the compose files (to make sure we don't attempt to pull them down from the docker registry)

Reply:
Yeah, I guess we would. But maybe for consistency they should be in a registry... what do you think?

Reply:
I think it would indeed be ideal if they are in a registry!

Reply:
Let's aim for that then! For ease of development I might keep building them locally, but we can upload them and take them out of source closer to launch.

Reply:
I think we might also need to remove tags from images when we do this (so that Inspect doesn't default to not pulling them from the registry).

Reply:
Having them in a registry has several benefits:
- Dockerfile != image. An image is immutable. A Dockerfile can result in different images over time e.g. it might install the latest version of a package. Or it might reference a base image which changes over time. Or it might no longer even be buildable one day.
- This works with k8s. On k8s all images will need to be in a registry which the cluster has access to.

We should discuss internally to AISI what our policy is on pushing to public registries.

I'd also say that whilst it is convenient to use `:latest` (or no tag specified, which implicitly uses latest) while developing, it is better practice to reference a particular version (i.e. tag) of your image from your compose.yaml once we've pushed the image to a registry. This will keep our evals stable/consistent until we explicitly reference a different tag. We can use semver for this.

Reply:
If we push them to a registry, it doesn't necessarily mean we shouldn't keep the Dockerfile in this repo though. They're useful for reference (without having to reverse engineer the docker image). We should clearly document that they're for reference only though.

Reply:
If we do end up keeping this `build.sh`, I think we should rename it to `build-images.sh`. In ctf-evals-template, this lived in the `images` dir so already had this context.

Reply:
>If we push them to a registry, it doesn't necessarily mean we shouldn't keep the Dockerfile in this repo though.

I guess some of the files within the `challenges/*/images` dir aren't strictly only for building images - they are also copied into the agent/default container at run time, so will definitely have to be kept in this repo.